File: NapTheOnline/ClientApp/src/app/share/view-model/news.model.ts
export class NewsModel {
    id: number;
    logo: string;
    name: string;
    description: string;
    dateCreated: string;
    typeId: number;

    // only UI
    friendlyName: string;

    public constructor(init?: Partial<NewsModel>) {
        Object.assign(this, init);
        this.typeId = 1;
    }
}

File: SourceCode/NapTheOnline/ViewModels/NewsModel.cs

using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace NapTheOnline.ViewModels
{
    public class NewsModel
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string id { get; set; }
        public string logo { get; set; }
        public string name { get; set; }
        public string friendlyname { get; set; }
        public string description { get; set; }
        public string datecreated { get; set; }
        public int typeid { get; set; }
    }
}
